include/MixerChannelView.h

```diff
@@ -57,6 +54,9 @@ class MixerChannelView : public QWidget
 	Q_PROPERTY(QColor strokeInnerActive READ strokeInnerActive WRITE setStrokeInnerActive)
 	Q_PROPERTY(QColor strokeInnerInactive READ strokeInnerInactive WRITE setStrokeInnerInactive)
 public:
+	static constexpr auto MixerChannelInnerBorderSize = 3;
+	static constexpr auto MixerChannelOuterBorderSize = 1;
```

Comment:
These could probably just be `InnerBorderSize` and `OuterBorderSize` now that they're inside the `MixerChannelView` class.